src/index.js

```diff
@@ -1,23 +1,25 @@
 import Plugin from '@swup/plugin';
 
-export default class PluginName extends Plugin {
-	name = 'PluginName';
+export default class SwupPluginNamePlugin extends Plugin {
```

Comment:
I'm not sure if you're aware, but maybe you got it covered already and the new strings are put in here on purpose. All of these strings (PluginName, [PluginName], etc.) are [used by CLI](https://github.com/swup/cli/blob/master/src/commands/create.js#L98-L108) to replace it with the correct format when creating a new plugin.

Reply:
Good to know, wasn't aware of that. I'll revert those and test this new version with the CLI.

Reply:
I've reverted all changes to name placeholders.